Tests/Unit/Repository/PhpcrRepositoryTest.php

```diff
@@ -11,15 +11,16 @@
 
 namespace Symfony\Cmf\Component\Resource\Tests\Unit\Repository;
 
+use PHPCR\NodeType\NodeTypeInterface;
+use PHPCR\PathNotFoundException;
 use Symfony\Cmf\Component\Resource\Repository\PhpcrRepository;
+use Symfony\Cmf\Component\Resource\Repository\Resource\PhpcrResource;
 
 class PhpcrRepositoryTest extends RepositoryTestCase
 {
     public function setUp()
     {
-        $this->session = $this->prophesize('PHPCR\SessionInterface');
-        $this->finder = $this->prophesize('DTL\Glob\FinderInterface');
-        $this->node = $this->prophesize('PHPCR\NodeInterface');
+        parent::setUp();
         $this->child1 = $this->prophesize('PHPCR\NodeInterface');
         $this->child2 = $this->prophesize('PHPCR\NodeInterface');
```

Comment:
ok.

Am 28.05.2016 um 21:16 schrieb dantleech:
>
> In Tests/Unit/Repository/PhpcrRepositoryTest.php
> <https://github.com/symfony-cmf/resource/pull/17#discussion_r64992833>:
>
> >  
> >  class PhpcrRepositoryTest extends RepositoryTestCase
> >  {
> >      public function setUp()
> >      {
> > -        $this->session = $this->prophesize('PHPCR\SessionInterface');
> > -        $this->finder = $this->prophesize('DTL\Glob\FinderInterface');
> > -        $this->node = $this->prophesize('PHPCR\NodeInterface');
> > +        parent::setUp();
> >          $this->child1 = $this->prophesize('PHPCR\NodeInterface');
> >          $this->child2 = $this->prophesize('PHPCR\NodeInterface');
>
> Then why not set them in the parent class? If they are of differnet
> types then they should be redeclared locally, they are different things.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/symfony-cmf/resource/pull/17/files/21f10728551210f9a65cdf1d90eb51c6885b449a#r64992833>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACxW6tecl4beVPfhl3dF4vD7HLmYp_buks5qGJSOgaJpZM4IjMjy>.
>